README.md

```diff
@@ -140,13 +142,15 @@ YV12/I420 as input format for csc/scaling/blending/rotation, etc. on Ubuntu 16.0
 (with kernel 4.10). The issue can be addressed with the kernel patch:
 WaEnableYV12BugFixInHalfSliceChicken7 [commit 0b71cea29fc29bbd8e9dd9c641fee6bd75f6827](https://cgit.freedesktop.org/drm-tip/commit/?id=0b71cea29fc29bbd8e9dd9c641fee6bd75f68274)
 
-3. CNL: HEVC encoding does not support P frame.
+3. KBL/CFL/CNL: HEVC encoding does not support P frame, detail see [issue#206](https://github.com/intel/media-driver/issues/206)/[issue#207](https://github.com/intel/media-driver/issues/207)/[issue#209](https://github.com/intel/media-driver/issues/209), patch in [PR#262](https://github.com/intel/media-driver/pull/262).
```

Comment:
Suggest to slightly reformulate: "KBL/CFL/CNL: HEVC encoding does not support P frame. For details see [issue#206](https://github.com/intel/media-driver/issues/206)/[issue#207](https://github.com/intel/media-driver/issues/207)/[issue#209](https://github.com/intel/media-driver/issues/209). Patches under review: [PR#262](https://github.com/intel/media-driver/pull/262"